lib/google_scraping/dashboard/keywords.ex

```diff
@@ -42,15 +43,73 @@ defmodule GoogleScraping.Dashboard.Keywords do
   Creates keywords list.
   """
   def create_keyword_list(keyword_list, user) do
```

Comment:
How about using the pattern matching to get the `user_id` directly here? so we dont need to `user.id` to ger the user id.
```suggestion
  def create_keyword_list(keyword_list, %User{id: user_id} = user) do
```